source/includes/read/distinct-async.py

```diff
@@ -0,0 +1,23 @@
+# start-distinct
+results = await restaurants.distinct("borough")
+
+async for restaurant in results:
```

Comment:
```suggestion
for restaurant in results:
```
`distinct()` returns a list.